package.json

```diff
@@ -48,7 +48,8 @@
     "build:storybook": "rm -Rf storybook-static && build-storybook -s ./public # TODO: we shouldn't need to remove, but Storybook 6.1 has a small bug and can't remove existing build automatically",
     "start:storybook-static": "serve storybook-static",
     "auto-changelog": "auto-changelog -u",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "audit": "cd scripts/lighthouse && node home-stats.js && xdg-open lhreport.html && cd ../../"
```

Comment:
Can you remove the `cd`? With node `./scripts/lighthouse/home-stats`?

Reply:
I have 2 choices here:
The results of the script (lhreport.html) will appear in the directory I use to run the script, so I can either let it on scripts/lighthouse or remove the cd for a clearer command but we'll have the results in the application root directory. What do you preafer ?

Reply:
You should store the report in a root directory that is ignored by git (eg "/audits" or smth), so yes I'd created the report in the directory where I run the script which is the most relevant.